src/core/model_config_utils.cc

```diff
@@ -648,108 +647,24 @@ GetNormalizedModelConfig(
   // If the model name is not given in the configuration, set if based
   // on the model path.
   const std::string model_name(BaseName(path));
-  if (config->name().empty()) {
-    config->set_name(model_name);
-  }
-
-  // FIXME need to check other Triton components on how they retrieve model
-  // config. The new workflow will let model backend contains the most updated
-  // config after the model is loaded, in other word, should always retrieve
-  // config with backend.Config().
-  // Autofill if requested...
-  if (autofill) {
-    std::unique_ptr<AutoFill> af;
-    RETURN_IF_ERROR(AutoFill::Create(
-        model_name, backend_config_map, std::string(path), *config, &af));
-    RETURN_IF_ERROR(af->Fix(config));
-
-    LOG_VERBOSE(1) << "autofilled config: " << config->DebugString();
-  }
-
-  // Fill backend if platform is set for non-custom backend
-  if (config->backend().empty() && !config->platform().empty()) {
-#ifdef TRITON_ENABLE_TENSORRT
-    if ((config->platform() == kTensorRTPlanPlatform)) {
-      config->set_backend(kTensorRTBackend);
-    }
-#endif  // TRITON_ENABLE_TENSORFLOW
-#ifdef TRITON_ENABLE_TENSORFLOW
-    if ((config->platform() == kTensorFlowGraphDefPlatform) ||
-        (config->platform() == kTensorFlowSavedModelPlatform)) {
-      config->set_backend(kTensorFlowBackend);
-    }
-#endif  // TRITON_ENABLE_TENSORFLOW
-#ifdef TRITON_ENABLE_ONNXRUNTIME
-    if (config->platform() == kOnnxRuntimeOnnxPlatform) {
-      config->set_backend(kOnnxRuntimeBackend);
-    }
-#endif  // TRITON_ENABLE_ONNXRUNTIME
-#ifdef TRITON_ENABLE_PYTORCH
-    if (config->platform() == kPyTorchLibTorchPlatform) {
-      config->set_backend(kPyTorchBackend);
-    }
-#endif  // TRITON_ENABLE_PYTORCH
-    // FIXME: "else if ()" other supported frameworks once they are ported
-    // to use backend API.
-  }
 
-  // FIXME: Add other supported frameworks once they are ported
-  // to use backend API.
-  // // Fill platform if backend is set for non-custom backend
-  // if (!config->backend().empty() && config->platform().empty()) {
-  //   // tensorflow can't be filled as platform is not unique
-  // }
-  if (!config->backend().empty() && config->platform().empty()) {
-#ifdef TRITON_ENABLE_ONNXRUNTIME
-    if (config->backend() == kOnnxRuntimeBackend) {
-      config->set_platform(kOnnxRuntimeOnnxPlatform);
-    }
-#endif  // TRITON_ENABLE_ONNXRUNTIME
-#ifdef TRITON_ENABLE_PYTORCH
-    if (config->backend() == kPyTorchBackend) {
-      config->set_platform(kPyTorchLibTorchPlatform);
-    }
-#endif  // TRITON_ENABLE_PYTORCH
-  }
+  // Server side autofill only tries to determine the backend of the model, and
+  // it is only for baackends known by Triton. Because extracting more detailed
```

Comment:
baackends->backends

The comment can be improved like:
Server-side autofill only set backend fields for the models that belong to limited backends for backwards-compatibility. See <link to the known backends>. Extracting detailed information is delegated to the backend implementation to  auto-complete.